### PR TITLE
auto-detect latest openshift-client-linux-4.x.y.tar.gz

### PR DESCRIPTION
**Pull request by drewandersonnz**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -6,13 +6,19 @@ if [ ! -f ./.git/config ]; then
 fi
 
 if [ "x${osv4client}" == "x" ]; then
+    # auto-detect latest openshift-client-linux-4.x.y.tar.gz
+    osv4clienturl="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/"
+    export osv4client=`curl -s -q ${osv4clienturl} \
+        | grep openshift-client-linux-4 | grep .tar.gz  \
```
Comment (georgettica) [inline build.sh:12]: change this to client-linux without the 4 to allow for next versions to allow for better upgrades

Comment (drewandersonnz) [reply build.sh:12]: @georgettica I'm thinking if we get to OpenShift 5.x then we'll hav bigger upgrade problems :-) This is intentional to stop it working if there is an OpenShift 5.x

Comment (georgettica) [reply build.sh:12]: ok, I thinks that is a pair point

**Pull request merged**